mod/kalvidpres/lang/en/kalvidpres.php

```diff
@@ -36,3 +36,4 @@
 $string['kalvidpres:addinstance'] = 'Add a Kaltura Video Presentation';
 $string['eventvideo_resource_viewed'] = 'Video resource viewed';
 $string['privacy:metadata'] = 'Kaltura video presentation plugin does not store any personal data.';
+$string['search:activity'] = 'Kaltura Video Presentation';
```

Comment:
hi @yedidiaklein,
thanks for your contribution, can you please change the text here to "Kaltura Media Assignment"?
thanks!